docs/02.API-REFERENCE.md

```diff
@@ -5348,14 +5346,25 @@ main (void)
   jerry_init (JERRY_INIT_EMPTY);
 
   static uint32_t save_literal_buffer[256];
+  static uint32_t snapshot_buffer[256];
   const jerry_char_t *code_for_literal_save_p = (const jerry_char_t *) "var obj = { a:'aa', bb:'Bb' }";
 
-  size_t literal_sizes = jerry_parse_and_save_literals (code_for_literal_save_p,
-                                                        strlen ((const char *) code_for_literal_save_p),
-                                                        false,
-                                                        save_literal_buffer,
-                                                        sizeof (save_literal_buffer) / sizeof (uint32_t),
-                                                        true);
+  jerry_value_t generate_result;
+  generate_result = jerry_generate_snapshot (NULL,
```

Comment:
Is it needed to split this into two statements? I've checked, `jerry_value_t generate_result = jerry_generate_snapshot (...` fits in the line length limit.

Reply:
It is not needed indeed, but I think it looks a bit better. I can change if you want. Both solutions are fine to me.

Reply:
Variables should be initialized at the time of their declaration whenever possible. This may be called a personal preference. It's no blocker but wanted to make a note.